hazelcast/cluster.py

```diff
@@ -91,6 +104,7 @@ def remove_listener(self, registration_id):
         return self._service.remove_listener(registration_id)
 
     def get_members(self, member_selector=None):
+        # type: (Callable[..., None]) -> List[MemberInfo]
```

Comment:
`member_selector` is a function of the type `(MemberInfo) -> bool`, can you fix the type hint?